### DISCUSSION TITLE
how depoly project to server

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#4097] [Q&A] [answered]
how to depoly project (for example test-apps/display-test-app) to server.
now i download the code from itwinjs-core and build (by rushjs),and run success in dev pc.
but how to depoly project to product server and run start:backend (node --max-http-header-size=16000 lib/backend/WebMain.js).

**Comment by pmconne** [comment, marked as answer]
display-test-app is for testing purposes only, running on a single machine. It is not designed for server deployment, nor to serve as an example or template for "real" applications.
If you intend to develop a browser-based app, consider starting [here](https://www.itwinjs.org/learning/tutorials/develop-web-viewer/).

**Reply by paprophet** [reply]
thank you!
